src/components/AssetInput/AssetInput.tsx

```diff
@@ -69,181 +59,42 @@ type AssetType =
-                    ])}
-                >
-                    <div className="tw-min-w-0 tw-pr-3 tw-pl-4 tw-flex tw-flex-auto tw-flex-col tw-items-start tw-justify-center tw-h-full">
-                        <span
-                            id={labelId}
-                            className={merge([
-                                "tw-max-w-full tw-text-black-100 tw-text-s tw-truncate dark:tw-text-white",
-                                (isOpen || isFocusVisible) && "tw-font-medium",
-                            ])}
-                        >
-                            {title}
-                        </span>
-                        <AssetSubline asset={asset} isLoading={isLoading} />
-                    </div>
-                    <div className="tw-p-4 tw-flex tw-flex-none tw-items-center tw-justify-center">
-                        <span
-                            className={merge([
-                                "tw-transition-transform",
-                                isOpen ? "tw-rotate-180 tw-text-black-90" : "tw-text-black-60",
-                            ])}
-                        >
-                            <IconCaretDown size={IconSize.Size16} />
-                        </span>
-                    </div>
-                </div>
-            </button>
-
-            <AnimatePresence>
-                {isOpen && (
-                    <motion.div
-                        className="tw-absolute tw-left-0 tw-w-full tw-overflow-hidden tw-box-border tw-p-0 tw-shadow-mid tw-list-none tw-m-0 tw-mt-2 tw-z-10"
-                        key={`asset-input-menu-${menuId}`}
-                        initial={{ height: 0 }}
-                        animate={{ height: "auto" }}
-                        exit={{ height: 0 }}
-                        transition={{ ease: [0.04, 0.62, 0.23, 0.98] }}
-                    >
-                        <FocusScope restoreFocus>
-                            <div {...overlayProps} ref={overlayRef}>
-                                <DismissButton onDismiss={() => menuState.close()} />
-                                <ActionMenu menuBlocks={actions} focus={focusStrategy} />
-                                <DismissButton onDismiss={() => menuState.close()} />
-                            </div>
-                        </FocusScope>
-                    </motion.div>
-                )}
-            </AnimatePresence>
-        </div>
-    );
-};
-
-export const AssetInput: FC<AssetInputProps> = ({
-    asset,
+export const AssetInput: FC<AssetProps> = ({
+    assets,
+    numberOfLocations,
     actions = [],
     size = AssetInputSize.Small,
     isLoading = false,
     onLibraryClick,
     onUploadClick,
+    onMultiAssetClick,
 }) => {
-    if (isLoading || asset) {
-        return <SelectedAsset asset={asset} size={size} actions={actions} isLoading={isLoading} />;
+    const availableAssets = assets || [];
+    const lengthAssets = availableAssets?.length || 0;
```

Comment:
I'd rename this to `assetsLength` because it sounds more intuitive to me, but you can keep your version if you prefer it

Reply:
done 👌🏻